src/DGmethods/balancelaw.jl

```diff
@@ -138,3 +138,7 @@ function create_diffstate(bl, grid, commtag=111)
 
   return diffstate
 end
+
+function calculate_dt(grid, model::BalanceLaw)
+    return 0.1
+end
```

Comment:
`BalanceLaw` is an abstract type -- I think this function should be empty?

Reply:
yeah we can do that instead